Lib/codeop.py

```diff
@@ -79,17 +80,22 @@ def _maybe_compile(compiler, source, filename, symbol):
     code = code1 = code2 = None
 
     try:
+        # Capture warnings only on the first call to avoid duplication.
```

Comment:
```suggestion
        # Allow warnings only on the first call to avoid duplication.
```
but see alternative.

Reply:
I went with the alternative of moving the comment to the ignored warnings.